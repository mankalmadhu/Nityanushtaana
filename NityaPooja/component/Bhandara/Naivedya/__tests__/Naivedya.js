import 'react-native';
import React from 'react';
import ನೈವೇದ್ಯ from '../Naivedya';

import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವಾ೦ಗ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Vidhana',()=>'ವಿಧಾನ');

test("ನೈವೇದ್ಯ" , () => {

expect(Renderer.create(
    <ನೈವೇದ್ಯ />
).toJSON()).toMatchSnapshot();

});