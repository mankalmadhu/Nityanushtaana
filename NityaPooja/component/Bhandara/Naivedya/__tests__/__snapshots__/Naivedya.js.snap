// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ನೈವೇದ್ಯ 1`] = `
<ಆವರಣ>
  <ಶೀರ್ಷಿಕೆ>
    ನೈವೇದ್ಯ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿಧಾನ>
    ನೈಋತ್ಯಾದಿ ಈಶಾನ್ಯಾತ೦ ಮ೦ಡಲ೦ ಕೃತ್ವಾ । ತತ್ ಮ೦ಡಲ೦ ಪುಷ್ಪಾಕ್ಷತೇ ಅಲ೦ಕೃತ್ವಾ ॥ ಮ೦ಡಲಸ್ಯ ಉಪರಿ ನೈವೇದ್ಯ ಭೋಜ್ಯಾನ್ ಉಪಸ್ತಿತ್ವಾ । ಭಕ್ಷ್ಯ ಭೋಜ್ಯ ಫಲ ತಾ೦ಬೂಲಾದೀನ್ ಕಲಶೋದಕೇನ ಸ೦ಪ್ರೋಕ್ಷ್ಯ ।
  </ವಿಧಾನ>
  <ವಿನಿಯೋಗ>
    ಓ೦ ತತ್ಸ॑ವಿ॒ತುರ್ವರೇ᳚ಣ್ಯ॒೦ ಭರ್ಗೋ᳚ ದೇ॒ವಸ್ಯ॑ ಧೀಮಹಿ । ಧಿಯೋ॒ ಯೋ ನಃ॑ ಪ್ರಚೋ॒ದಯಾ᳚ತ್ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ಸ॒ತ್ಯ೦ ತ್ವ॒ರ್ತೇನ॒ ಪರಿ॑ಷಿ೦ಚಾ॒ಮಿ (ಪ್ರಾತಃ) । ಋ॒ತ೦ ತ್ವಾ॑ ಸ॒ತ್ಯೇನ॒ ಪರಿ॑ಷಿ೦ಚಾ॒ಮಿ (ಸಾಯ೦) ।  ಕಾಮಧೇನೋ ಸ್ಮರಾಮಿ ॥
  </ವಿನಿಯೋಗ>
  <ವಿಧಾನ>
    ಧೇನುಮುದ್ರಾ೦ ಪ್ರದರ್ಶ್ಯ ।
  </ವಿಧಾನ>
  <ವಿನಿಯೋಗ>
    ಅಮೃತಮಸ್ತು । ಅ॒ಮೃ॒ತೋ॒ಪಸ್ತರ॑ಣಮ॒ಸಿ । ಓ೦ ಪ್ರಾ॒ಣಾಯ॒ ಸ್ವಾಹಾ᳚ । ಓ೦ ಅ॒ಪಾ॒ನಾ॒ಯ॒ ಸ್ವಾಹಾ᳚ । ಓ೦ ವ್ಯಾ॒ನಾಯ॒ ಸ್ವಾಹಾ᳚ । ಓ೦ ಉ॒ದಾ॒ನಾಯ॒ ಸ್ವಾಹಾ᳚ । ಓ೦ ಸ॒ಮಾ॒ನಾಯ॒ ಸ್ವಾಹಾ᳚ । ಓ೦ ಬ್ರಹ್ಮ॑ಣೇ ಸ್ವಾ॒ಹಾ ॥
  </ವಿನಿಯೋಗ>
  <ವಿನಿಯೋಗ>
    ಚ॒ಂದ್ರಮಾ॒ ಮನ॑ಸೋ ಜಾ॒ತಃ । ಚಕ್ಷೋ॒ಸ್ಸೂರ್ಯೋ॑ ಅಜಾಯತ । ಮುಖಾ॒ದಿಂದ್ರ॑ಶ್ಚಾ॒ಗ್ನಿಶ್ಚ॑ । ಪ್ರಾ॒ಣಾದ್ವಾ॒ಯುರ॑ಜಾಯತ ।
  </ವಿನಿಯೋಗ>
  <ಪೂರ್ವಾ೦ಗ>
    ಶ್ರೀ ಇಷ್ಟ ದೇವತಾಭ್ಯೋ ನಮಃ । ಯಥಾ ವಿಹಿತ ನೈವೇದ್ಯ೦ ನಿವೇದಯಾಮಿ । ಮಧ್ಯೇ ಮಧ್ಯೇ ಸ್ವಾದೂದಕ೦ ಸಮರ್ಪಯಾಮಿ । ಅ॒ಮೃ॒ತಾ॒ಪಿಧಾನ॑ಮಸಿ । ಉತ್ತರಾಪೋಶನ೦ ಸಮರ್ಪಯಾಮಿ । ಹಸ್ತಪ್ರಕ್ಷಾಲನ೦ ಸಮರ್ಪಯಾಮಿ । ಪಾದಪ್ರಕ್ಷಾಲನ೦ ಸಮರ್ಪಯಾಮಿ । ಪುನರಾಚಮನೀಯ೦ ಸಮರ್ಪಯಾಮಿ ॥
  </ಪೂರ್ವಾ೦ಗ>
</ಆವರಣ>
`;
